santosfamilyfoundation/RuinaMain
# [RUIN-243] Clean Up UI from Native Base Transition

# Description

Catching all the small UI changes that were obvious after the Native Base merge. One big fix is the export screen. After merging, the export screen was broken because a button used UI Kitten. That code has been fixed and replaced. There is now a start new report button. However, the data is not being reset when pressed. This is something that should be investigate in the future because it might cause data to be overwritten.

# Testing

1. Start a report
2. Click on all the buttons to make sure that nothing is still broken.
3. Click on the export button.
4. Check that on each of the export screens there is a button to start a new report.
